source/frontend/src/features/contacts/hooks/useConceptPersonDetail.ts

```diff
@@ -0,0 +1,57 @@
+import axios, { AxiosError } from 'axios';
+import { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { hideLoading, showLoading } from 'react-redux-loading-bar';
+
+import { useApiContacts } from '@/hooks/pims-api/useApiContacts';
+import { useApiRequestWrapper } from '@/hooks/util/useApiRequestWrapper';
+import { IApiError } from '@/interfaces/IApiError';
+import { Api_Person } from '@/models/api/Person';
+import { logError } from '@/store/slices/network/networkSlice';
+
+/**
+ * hook that fetches the person given the person's id.
+ * @param personId
+ */
+export const useConceptPersonDetail = (personId?: number | null) => {
```

Comment:
@buddy326 getting closer, but this is more or less the same format as the existing usePersonDetail, which is a really old pattern. Please update this to be more like our *Repository hooks. Ie. useTakesRepository.

Reply:
Updated! I will remove the concept hooks as soon as the new repository hooks get approved.